docker-entrypoint.sh

```diff
@@ -1,6 +1,37 @@
 #!/bin/bash
 set -euo pipefail
 
+# usage: file_env VAR [DEFAULT]
+#    ie: file_env 'XYZ_DB_PASSWORD' 'example'
+# (will allow for "$XYZ_DB_PASSWORD_FILE" to fill in the value of
+#  "$XYZ_DB_PASSWORD" from a file, especially for Docker's secrets feature)
+file_env() {
+	local var="$1"
+	local fileVar="${var}_FILE"
+	local def="${2:-}"
+	if [ "${!var:-}" ] && [ "${!fileVar:-}" ]; then
+		mysql_error "Both $var and $fileVar are set (but are exclusive)"
```

Comment:
`mysql_error` function is not defined. Maybe just an echo?

Reply:
copy/paste will get you every time 😄

Copied from https://github.com/docker-library/mysql/blob/master/8.0/docker-entrypoint.sh#L16
do we want all the information at the referenced https://github.com/docker-library/mysql/blob/master/8.0/docker-entrypoint.sh#L8 ?

Reply:
Arf, that looks a bit too much... the following might be sufficient right?
```suggestion
		printf 'Both %s and %s are set (but are exclusive)' "$var" "$fileVar"
```